.pipelines/ci.yml

```diff
@@ -1,20 +1,24 @@
 # Azure DevOps Pipeline running CI
 trigger:
-- master
+  branches:
+    include:
+    - master
+  paths:
+    exclude:
+    - docs/*
+
+pr:
+  branches:
+    include:
+    - master
+  paths:
+    exclude:
+    - docs/*
 
 variables:
 - template: vars.yml
 
 jobs:
-- job: Golang_Code_Lint
-  pool: ARO-CI
-  steps:
-  - template: ./templates/template-checkout.yml
-  - script: |
-      set -xe
-      make lint-go
```

Comment:
We no longer seem to run lint-go in CI. I think we still need it.

Reply:
Fixed, thanks for spotting it.